x-pack/plugins/ml/public/application/components/anomalies_table/links_menu.tsx

```diff
@@ -75,15 +76,29 @@ export const LinksMenuUI = (props: LinksMenuProps) => {
 
   const getMapsLink = async (anomaly: AnomaliesTableRecord) => {
     const initialLayers = getInitialAnomaliesLayers(anomaly.jobId);
-    const anomalyBucketStart = moment(anomaly.time).toISOString();
+    const anomalyBucketStartMoment = moment(anomaly.time).tz(getDateFormatTz());
+    const anomalyBucketStart = anomalyBucketStartMoment.toISOString();
+    const anomalyBucketEnd = anomalyBucketStartMoment
```

Comment:
You should subtract 1ms here so that Maps shows only a single bucket and not the start of the next one.

Reply:
Added in 92b01f3fc4a9280729f738b05e3b87f8300317d7